package exception;

public class Exception01_1 {
	public static void main(String[] args) {
		System.out.println("100 ÷ 0 は？");
		int result = 100 / 0;
		System.out.println("計算結果" + result);
		System.out.println("プログラミング終了");
	}

}
